main/monmon.rb

```diff
@@ -1,14 +1,15 @@
 require 'csv'
 
-balance = Hash.new
+balance = Hash.new(0)
 
-CSV.foreach("input.csv", headers: true) do |row|
-  puts("#{row['Type']}: #{row['Name']}:  #{row['Currency']}\t - #{row['Amount']}  #{row['Currency']}")
-  if balance.include?(row['Currency'])
-    balance[row['Currency']] += row['Amount'].to_i
+CSV.foreach("input.csv", :headers => true, :header_converters => :symbol) do |row|
```

Comment:
`=>` is an old syntax to specify hashes.
Use the new ruby syntax for hashes: `headers: true, header_converters: :symbol`.

Nevertheless, `=>` is used even now, but in special cases, then keys are strings, for example: `'key' => value`.

Reply:
Also prefer `'input.csv'` over `"input.csv"` if interpolation is not used.

Reply:
Ok. I will correct in the near future